src/app/features/game/game.component.html

```diff
@@ -0,0 +1,54 @@
+<ng-template [ngIf]="gameData">
+  <h1>{{ gameData.title }}</h1>
+  <div>Platform: {{ gameData.platform }}</div>
+  <div>
+    In {{ gameData.saved_by.length }}
+    {{ gameData.saved_by.length === 1 ? 'collection' : 'collections' }}
+  </div>
+  <p-button
+    *ngIf="!inUserCollection"
+    class="p-button-success"
+    label="Add to your collection"
+    (click)="addToCollection()"></p-button>
+  <p-button
+    *ngIf="inUserCollection"
+    class="p-button-danger"
+    label="Remove from your collection"
+    (click)="removeFromCollection()"></p-button>
+  <p-divider></p-divider>
+  <form
+    *ngIf="gameForm"
+    [formGroup]="gameForm"
+    [ngClass]="{ 'form-group': true, disabled: !inUserCollection }">
+    <h2>Your Details</h2>
+    <div class="form-selection">
+      <span class="form-container">
+        <h3>Media form:</h3>
```

Comment:
hierarchy 👍
accessibility (a11y) is very important. wcag 2 is the standard https://www.w3.org/WAI/standards-guidelines/wcag/

there are checklist https://www.a11yproject.com/checklist/

you get the feeling for 80% very easily if you invest some time. take your time 🙌

```
<img src="elephant.jpg" alt="Elephant stands in frotn of four trees ..." />
<img src="http://www.separator.jpg" alt="" />
```